qiita_pet/templates/redbiom.html

```diff
@@ -153,21 +158,83 @@
+  </small>
+  <div id="redbiom-help" class="collapse">
     <br/>
     We have 3 search options:
     <ul>
-      <li><b>Metadata</b>: The search will be on the full metadata and will return any ocurrances of the search.</li>
-      <li><b>Feature</b>: The seach will be on all the features, in specific: OTU ids for close reference and exact sequences for deblur</li>
-      <li><b>Taxon</b>: The search will be only on close reference and based on the taxonomies available</li>
+      <li>
+        <b>Metadata</b><br/>
+        The search will be on the full metadata.
+        <br/><br/>
+        The metadata search engine uses natural language processing to search for
+        word stems within a samples metadata. A word stem disregards modifiers and
+        plurals, so for instance, a search for "antibiotics" will actually perform
+        a search for "antibiot". Similarly, a search for "crying" will actually
+        search for "cry". The words specified can be combined with set-based
+        operations, so for instance, a search for "antibiotics &amp; crying" will
+        obtain the set of samples in which each sample has "antibiot" in its
+        metadata as well as "cry". N.B., the specific category in which a stem is
+        found is not assured to be the same, "antibiot" could be in one category
+        and "cry" in another. A set intersection can be performed with "&amp;", a
+        union with "|" and a difference with "-".
+        <br/><br/>
+        In addition to the stem-based search, value based searches can also be a
+        applied. These use a Python-like grammar and allow for a rich set of
+        comparisons to be performed based on a metadata category of interest. For
+        example, "where qiita_study_id == 10317" will find all samples which have
+        the qiita_study_id metadata category, and in which the value for that
+        sample is "10317."
+        <br/><br/>
+        Examples:
+        <br/>
+        <ul>
+          <li>
+            Find all samples in which the word infant exists, as well as antibiotics,
+            where the infants are under a certain number of days old:
+            "infant &amp; antibiotics where age_days < 30"
+          </li>
+          <li>
+            Find all samples only beloging to the EMP in which the ph in under 7:
+            "where ph < 7 &amp; emp_release1 == True"
+          </li>
+        </ul>
+      </li>
+      <li>
+        <b>Feature</b>:<br/>
+        The seach will be on all the features, in specific: OTU ids for close reference and exact sequences for deblur.
+        <br/><br/>
+        Examples:
+        <br/>
+        <ul>
+          <li>
+            Find all samples in which the GreenGenes feature 4479944 is found: "4479944"
+          </li>
+        </ul>
+      </li>
+      <li><b>Taxon</b>:<br/>
+        The search will be only on closed reference and based on the taxonomies available. Note that you need to use the GreenGenes nomenclature.
```

Comment:
Same here

"...nomenclature of a rank prefix and taxon (e.g., o__Clostridiales for order level Clostridiales)."

Reply:
I would also remind folks that there are dual underscores between rank and name. 

so maybe:
```
The search will be only on closed reference and based on the taxonomies available. 
Only exact matches are returned. Note that currently only the Greengenes taxonomy is 
searchable, and that it requires nomenclature of a rank prefix, two underscores, and then the
name (e.g., o__Clostridiales for order level Clostridiales).
```